homeassistant/components/vicare/types.py

```diff
@@ -64,6 +64,55 @@ def from_ha_preset(
 }
 
 
+class VentilationMode(enum.StrEnum):
+    """ViCare ventilation modes."""
+
+    PERMANENT = "permanent"  # on, speed controlled by program (levelOne-levelFour)
+    VENTILATION = "ventilation"  # activated by schedule
+    SENSOR_DRIVEN = "sensorDriven"  # activated by schedule, override by sensor
+    SENSOR_OVERRIDE = "sensorOverride"  # activated by sensor
+
+    @staticmethod
+    def to_ha_mode(mode: str | None) -> str | None:
+        """Return the mapped Home Assistant mode for the ViCare ventilation mode."""
+        if mode:
+            try:
+                ventilation_mode = VentilationMode(mode)
+            except ValueError:
+                # ignore unsupported / unmapped modes
+                return None
+            return VICARE_TO_HA_MODE_VENTILATION.get(ventilation_mode) if mode else None
+        return None
+
+    @staticmethod
+    def from_ha_mode(ha_mode: str | None) -> str | None:
+        """Return the mapped ViCare ventilation mode for the Home Assistant mode."""
+        for mode in VentilationMode:
+            if VICARE_TO_HA_MODE_VENTILATION.get(VentilationMode(mode)) == ha_mode:
+                return mode
```

Comment:
Change to two lines in 
`if mode == ha_mode:
return VICARE_TO_HA_MODE_VENTILATION.get(mode)`

Reply:
There is a +- sign in the editor where you can directly do(suggest) the changes in the code.

Reply:
I can´t change the code directly in your pull request.

Reply:
You're not allowed to do this?

```suggestion
            if VICARE_TO_HA_MODE_VENTILATION.get(VentilationMode(mode)) == anything:
                return mode
```